codechain/run_node.rs

```diff
@@ -111,8 +111,15 @@ fn stratum_start(cfg: &StratumConfig, miner: &Arc<Miner>, client: Arc<Client>) -
     }
 }
 
-fn new_miner(config: &config::Config, scheme: &Scheme, ap: Arc<AccountProvider>) -> Result<Arc<Miner>, String> {
+fn new_miner(
+    timer_loop: &Arc<TimerLoop>,
```

Comment:
Do you have a reason to pass `&Arc<TimerLoop>` instead of `&TimerLoop`?

Reply:
So it's possible to convert `Arc<TimerLoop>` into `&TimerLoop`?

Reply:
Fixed.